inst/extdata/fd_nested_subplots.csv

```diff
@@ -0,0 +1,19 @@
+﻿Subplot,Nested_subplot,Nested_subplot_area_m2
+E,1,1
+W,1,1
+E,2,1
+W,2,1
+E,3,1
+W,3,1
+E,4,1
+W,4,1
+E,5,1
+W,5,1
+E,6,1
+W,6,1
+E,7,1
+W,7,1
+E,8,1
+W,8,1
+E,9,1
+W,9,1

```

Comment:
Unless I'm missing something, shouldn't there be one of these for every Replicate x Plot? And therefore, shouldn't Replicate and Plot be included as columns in here?

Finally, perhaps the most nit-picky comment I have ever contributed to anything ever, but perhaps it might make more sense to alter the sorting so the smallest subdivision (`Nested_subplot`) varies the fastest?

```
Subplot, Nested_subplot
E,1
E,2
E,3
E,4
...
W,1
W,2
W,3
```
To me, this makes the hierarchy a little clearer.

Reply:
>shouldn't there be one of these for every Replicate x Plot? 

Yes, but they're all identical, right? So I thought just have this one table, and we can duplicate across plots as necessary.

Reply:
>alter the sorting

If it makes you happy 🤷‍♂️

Reply:
>Yes, but they're all identical, right? So I thought just have this one table, and we can duplicate across plots as necessary.

This is correct, but you're probably right--it's less confusing just to expand the table.